LICENSES/LicenseRef-MIT-OA.txt

```diff
@@ -1,6 +1,6 @@
 MIT License
-Copyright (c) 2021-2022 Oxhead Alpha
-Copyright (c) 2019-2021 Tocqueville Group
+Copyright (c) 2022-2022 Oxhead Alpha
+Copyright (c) 2019-2022 Tocqueville Group
```

Comment:
Is this change right? I'm also not sure, but it seemed fine as it was.

Reply:
Yeah, `Copyright (c) 2019-2021 Tocqueville Group` means that this code belonged to Tocqueville Group during 2019-2021. But they don't own the edits after that (because they don't pay for future code maintenance), so that part of license header remains unmodified.

Hmm, if Danger bot catches you here, maybe it would be cool to update the rules respectively in some separate MR (and for now let the warning be).

Reply:
Yeah, let's fix these years in the next PR (and in the next year :smile: )